client/my-sites/plugins/plugin-management-v2/plugin-row-formatter/index.tsx

```diff
@@ -1,19 +1,47 @@
 import { Gridicon, Button } from '@automattic/components';
+import { useSelector } from 'react-redux';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import PluginActivateToggle from 'calypso/my-sites/plugins/plugin-activate-toggle';
+import PluginAutoupdateToggle from 'calypso/my-sites/plugins/plugin-autoupdate-toggle';
+import { isMarketplaceProduct } from 'calypso/state/products-list/selectors';
+import { getAllowedPluginActions } from '../utils/get-allowed-plugin-actions';
 import type { Plugin } from '../types';
-import type { ReactChild, ReactElement } from 'react';
+import type { SiteData } from 'calypso/state/ui/selectors/site-data';
+import type { MomentInput } from 'moment';
+import type { ReactElement, ReactChild } from 'react';
 
 import './style.scss';
 
 interface Props {
 	item: Plugin;
 	columnKey: string;
+	selectedSite: SiteData;
+	isSmallScreen?: boolean;
 }
 
-export default function PluginRowFormatter( { item, columnKey }: Props ): ReactElement | any {
+export default function PluginRowFormatter( {
+	item,
+	columnKey,
+	selectedSite,
+	isSmallScreen,
+}: Props ): ReactElement | any {
 	const PluginDetailsButton = ( props: { className: string; children: ReactChild } ) => {
 		return <Button borderless compact href={ `/plugins/${ item.slug }` } { ...props } />;
 	};
 
+	const moment = useLocalizedMoment();
+	const state = useSelector( ( state ) => state );
+
+	const ago = ( date: MomentInput ) => {
+		return moment.utc( date, 'YYYY-MM-DD hh:mma' ).fromNow();
+	};
```

Comment:
I'm curious if this would work if the user's locale is not English–do you know if it is implemented in the same way in WordPress.com?

Reply:
Yes, it is implemented the exact same wat in WP.com